### ISSUE TITLE
Crashes when user inputs wrong password

### ISSUE THREAD

**Issue by dahnny012** [dahnny012/respot#12]
MongoError: Not a valid BCrypt hash.

Assign to wenny

**Comment by dahnny012** [comment]
Im going to purge the mongo db

**Comment by dahnny012** [comment]
Tested all logging in scenarios, seems fine.
Going to keep this up.

**Comment by leewc** [comment]
Yeah when I saw not a valid BCrypt hash it's most likely the older account had no encryption there. Good that purging the DB fixed it. Voting to close this issue, and then reopening it if it happens again.
